dao.js

```diff
@@ -0,0 +1,41 @@
+var pg = require('pg');

```

Comment:
Can you change all `var`s to either `const` or `let`? Although in this case they can all be `const` since they don't get reassigned. Also, can we use 2 space indent for any javascript/typescript files to maintain consistency?